components/team-info/sections/TeamInfo.vue

```diff
@@ -30,8 +30,14 @@
       </div>
     </div>
     <div class="flex justify-center items-center mt-3">
-      <n-button strong ghost type="primary" class="w-full md:w-1/2 h-20" @click="$router.push('/project/login')">
-        Project section
+      <n-button
+        strong
+        round
+        type="primary"
+        class="w-1/2 md:w-1/2 h-20 text-xl rounded-lg"
```

Comment:
Width has no effect on smaller devices. Please change it to full width![image](https://github.com/fiit-tp7-2023/frontend/assets/64609060/a5d4ed6e-9add-4265-9fdf-5eb3114b7ee8)